### PR TITLE
Enhanced object literals don't have arrows

### PR DESCRIPTION
**Pull request by jowenjowen**
Fixes a bug in serializeFunc. I added several unit tests until I could replicate the behavior. I think the extra unit tests are useful, but feel free to delete or alter any of them

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jowenjowen** [comment]
If you prefer a single commit, I can rework things

File: index.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ var PLACE_HOLDER_REGEXP = new RegExp('"@__(F|R|D|M|S)-' + UID + '-(\\d+)__@"', '
 
 var IS_NATIVE_CODE_REGEXP = /\{\s*\[native code\]\s*\}/g;
 var IS_PURE_FUNCTION = /function.*?\(/;
+var IS_ARROW_FUNCTION = /.+?=>.+?/;
```
Comment (okuryu) [inline index.js]: Why did you use `+?` not `*?`? I prefer `*?` like the above line if you don't have any reasons.

Comment (jowenjowen) [reply index.js]: Great catch! Yes, *? is much better. Can you make the change or do you want me to do it?

Comment (okuryu) [reply index.js]: Please make the change into your branch.

Comment (jowenjowen) [reply index.js]: Done

**Comment by okuryu** [comment]
Thanks!

**Comment by okuryu** [comment]
published `serialize-javascript@1.8.0`. Thanks!

**Pull request merged**